webui/react/src/utils/shortcut.ts

```diff
@@ -20,6 +24,32 @@ export const shortcutToString = (shortcut: KeyboardShortcut): string => {
   shortcut.meta && s.push(os.includes('Mac') ? 'Cmd' : os.includes('Win') ? 'Win' : 'Super');
   shortcut.shift && s.push('Shift');
   shortcut.alt && s.push('Alt');
-  shortcut.key && s.push(shortcut.key);
+  shortcut.key &&
+    !['Control', 'Meta', 'Alt', 'Shift'].includes(shortcut.key) &&
+    s.push(shortcut.key);
   return s.join(' + ');
 };
+
+export const formatKey = (code: string, key: string): string => {
```

Comment:
- Spacebar is displayed as ` `, should probably be `Space`
- I think 'Control', 'Meta', 'Alt', 'Shift' need to be excluded from `.toUpperCase()` because that's breaking the `.includes` check in `matchesShortcut`
![Screen Shot 2023-06-29 at 3 16 55 PM](https://github.com/determined-ai/determined/assets/97752292/f56235cb-cf2f-45bc-b9d1-16685c6143e3)
![Screen Shot 2023-06-29 at 3 18 11 PM](https://github.com/determined-ai/determined/assets/97752292/7a5ca491-fc88-4e7c-acc5-eecac17166f5)

Reply:
Good catch, I missed that. It should be fixed by now.